stellar-multitenant/internal/provisioning/manager.go

```diff
@@ -72,14 +74,24 @@ func (m *Manager) ProvisionNewTenant(
 		userEmail:     userEmail,
 		orgName:       organizationName,
 		networkType:   networkType,
+		uiBaseURL:     uiBaseUrl,
 	}
 
 	log.Ctx(ctx).Infof("adding tenant %s", name)
-	t, provisionErr := m.provisionTenant(ctx, pt)
+	t, tenantSchemaDSN, provisionErr := m.provisionTenant(ctx, pt)
```

Comment:
FYI: you don't need to return the DSN from provisionTenant, since you can use the tenantManager's `GetDSNForTenant` method to get that.

Reply:
sure sg, you're saying we would prefer to minimize the number of method return values right?

Reply:
Yes, that'd be a bit cleaner IMO

Reply:
This was addressed